app/src/main/res/values-zh-rTW/help.xml

```diff
@@ -56,26 +56,21 @@
         會失去你的Monero，其他人可以透過這個地址發送Monero到你的錢包。
         <h2>記憶種子碼</h2>
         這是你唯一需要用來回復錢包的資訊並且可以用來取用錢包的完全權限。妥善保護好這份種子碼是相當重要的事情，因為它可以給予<em>任何人</em>對你的錢包操作的權限! 如果你還尚未在安全的地方抄寫保存這份種子碼，請盡速進行。
-        <h2>Wallet Files Recovery Password</h2>
-        Make sure you write this password down. If you reset your device or uninstall the app, you
-        will need it to access your wallet again.<br/>
-        <h3>CrAzYpass</h3>
-        If the password displayed here is 52 alphanumeric characters in groups of 4 - congratulations!
-        Your wallet files are secured with a 256-bit key generated by your device&apos;s security
-        features based on the passphrase you chose (on creation or by changing it). This makes it
-        extremenly difficult to hack!<br/>
-        This feature is mandatory for all newly created wallets.
-        <h3>Legacy Password</h3>
-        If you see your passphrase here, your wallet files are not as secure as when using
-        a CrAzYpass. To fix this, simply select \"Change Passphrase\" from the menu. After entering
-        a new passphrase (maybe even the same one as before) the app will generate a CrAzYpass for
-        you and secure your wallet files with it. Write it down!
-        <h3>CrAzYpass wallets</h3>
-        If you ever need to reinstall Monerujo (for example after resetting your phone or switching
-        to a new one) or you want to use your wallet files on a different device or PC, you have to
-        use this Recovery Password in order to access your wallet again.<br/>
-        By selecting \"Change Passphrase\" from the menu, you can choose another passphrase. Beware
-        that this will generate a new Recovery Password. Write it down!
+        <h2>錢包檔回復密碼</h2>
+        請務必抄寫下此份密碼。當你重置手機或重新安裝APP時，你將會需要用它來重新存取錢包檔。<br/>
+        <h3>CrAzYpass加密</h3>
+        若你的密碼顯示為4組的52英數字元，恭喜你! 
```

Comment:
there is an extra blank at the end of this line.

Reply:
OK, fixed.